mappings/net/minecraft/world/biome/BiomeParticleConfig.mapping

```diff
@@ -1,5 +1,8 @@
 CLASS net/minecraft/class_4761 net/minecraft/world/biome/BiomeParticleConfig
 	FIELD field_22035 chance F
 	FIELD field_24675 CODEC Lcom/mojang/serialization/Codec;
+	FIELD field_24676 particleType Lnet/minecraft/class_2394;
+	METHOD <init> (Lnet/minecraft/class_2394;F)V
+		ARG 1 particleType
```

Comment:
```suggestion
	FIELD field_24676 particle Lnet/minecraft/class_2394;
	METHOD <init> (Lnet/minecraft/class_2394;F)V
		ARG 1 particle
```
returns a particle effect (with the getter por favor)

Reply:
Fixed in 19ea7d3d586be3cecd60f0f8dd730aca5d634efc.